revad/templates/configmap.yaml

```
@@ -1,3 +1,4 @@
+{{- if not .Values.cfgmapName }}
 {{- if .Values.configFiles }}
```

@jimil749 you can combine into a single line:
```suggestion
{{- if not .Values.cfgmapName and .Values.configFiles }}
```

---

Done